csrc/compute_at_map.cpp

```diff
@@ -153,11 +153,13 @@ bool IterDomainGraph::exprsMap(
 
     auto extent_0_match = extent_0o->sameAs(extent_1o) ||
         (extent_0o->isConstInt() && extent_1o->isConstInt() &&
-         extent_0o->evaluate() == extent_1o->evaluate());
+         extent_0o->evaluate().as<int64_t>() ==
```

Comment:
One question that's still unclear to me: Is this `as<T>()` strictly required?

Reply:
Not strictly `as<T>()`. Another alternative is `(extent_0o->evaluate() == extent_1o->evaluate())->as<bool>()`. But we have to make sure what the `&&` get is a `bool` instead of a `PolymorphicValue`.